### DISCUSSION TITLE
Kafka pods restart randomly

### DISCUSSION THREAD

**Discussion by iqzirr** [strimzi/strimzi-kafka-operator#6683] [Q&A]
![Untitled](https://user-images.githubusercontent.com/70002495/163317377-bf286c85-213f-4fa4-9412-50ea87bbe84b.png)

Hi Experts,

We're having a problem with our Kafka cluster, sometimes, some of the pods are randomly restarted, is this a normal behavior? or this is because of the CSI that we use, We're using Longhorn for my K8s storage for now.

**Comment by scholzj** [comment]
You will need to check why they are restarted. You also need to be specific about which pods are you talking about. The Cluster Operator sometimes restarts because of OoM - that is a known issue, it does not cause any general issues. That is one of the pods in your screenshot showing some restarts.
